New package: IntraManagementSoftware&Tech.FullMonteMSP version 3.7.25112.1

noyard opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/268249)

noyard committed:
New package: IntraManagementSoftware&Tech.FullMonteMSP version 3.7.25112.1

noyard commented:
> Hello @noyard,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

This is because Barbecana was purchased by Intra Management Software and technologies inc. See [https://www.barbecana.com/wp-content/uploads/2024/11/[Barbecana-Press-Release-11.1.24-Final-.pdf](https://www.barbecana.com/wp-content/uploads/2024/11/Barbecana-Press-Release-11.1.24-Final-.pdf)](https://www.barbecana.com/wp-content/uploads/2024/11/%5BBarbecana-Press-Release-11.1.24-Final-.pdf%5D(https://www.barbecana.com/wp-content/uploads/2024/11/Barbecana-Press-Release-11.1.24-Final-.pdf))

They have rebranded the installs but not the website. The URLs are good.

noyard commented:
> Hello @noyard,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain
> <!-- Policy app identification https://img.shields.io/static/v1?label=PullRequestIssueManagement. -->

See above ----,^^^^^

ItzLevvie commented:
@wingetbot run

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -2147012889 2025-06-30 03:51:02.870 [CORE] DeliveryOptimization error: 0x80072ee7, extended error: 0x00000000 2025-06-30 03:51:02.963 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!549157EB: (caller: 54915E1C) Exception(1) tid(11f0) 80072EE7  2025-06-30 03:51:15.177 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!5490F0BF: (caller: 54910E2B) Exception(2) tid(11f0) 80072EE7     Msg:[InternetOpenUrl() failed.]  2025-06-30 03:51:15.240 [CLI ] Failed to download, waiting a bit and retry. Url: https://www.barbecana.com/_downloads/FullMonteMSP37.exe 2025-06-30 03:51:36.618 [CORE] DeliveryOptimization error: 0x80072ee7, extended error: 0x00000000 2025-06-30 03:51:36.618 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!549157EB: (caller: 54915E1C) Exception(3) tid(11f0) 80072EE7  2025-06-30 03:51:48.692 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!5490F0BF: (caller: 54910E2B) Exception(4) tid(11f0) 80072EE7     Msg:[InternetOpenUrl() failed.] 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 00000000 | 0 | -4294967296 | ERROR_SUCCESS | The operation completed successfully. | 
 | 80072EE7 | -2147012889 | 2147954407 | ERROR_INTERNET_NAME_NOT_RESOLVED | The server name or address could not be resolved. | 
 | 80072EE7 | -2147012889 | 2147954407 | ERROR_INTERNET_NAME_NOT_RESOLVED | The server name or address could not be resolved. | 


(Automated response - build 1078.)

noyard commented:
> Automatic Validation ended with:
> 
> > Installation failed with exit code -2147012889 2025-06-30 03:51:02.870 [CORE] DeliveryOptimization error: 0x80072ee7, extended error: 0x00000000 2025-06-30 03:51:02.963 [FAIL] D:\a_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!549157EB: (caller: 54915E1C) Exception(1) tid(11f0) 80072EE7  2025-06-30 03:51:15.177 [FAIL] D:\a_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!5490F0BF: (caller: 54910E2B) Exception(2) tid(11f0) 80072EE7     Msg:[InternetOpenUrl() failed.]  2025-06-30 03:51:15.240 [CLI ] Failed to download, waiting a bit and retry. Url: https://www.barbecana.com/_downloads/FullMonteMSP37.exe 2025-06-30 03:51:36.618 [CORE] DeliveryOptimization error: 0x80072ee7, extended error: 0x00000000 2025-06-30 03:51:36.618 [FAIL] D:\a_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!549157EB: (caller: 54915E1C) Exception(3) tid(11f0) 80072EE7  2025-06-30 03:51:48.692 [FAIL] D:\a_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!5490F0BF: (caller: 54910E2B) Exception(4) tid(11f0) 80072EE7     Msg:[InternetOpenUrl() failed.]
> 
> Hex	Dec	Inverted Dec	Symbol	Description
> 00000000	0	-4294967296	ERROR_SUCCESS	The operation completed successfully.
> 80072EE7	-2147012889	2147954407	ERROR_INTERNET_NAME_NOT_RESOLVED	The server name or address could not be resolved.
> 80072EE7	-2147012889	2147954407	ERROR_INTERNET_NAME_NOT_RESOLVED	The server name or address could not be resolved.
> (Automated response - build 1078.)

Retried package install using manifest and did not receive error. 
![image](https://github.com/user-attachments/assets/d1dc5759-5d5c-4f81-9311-1b7eb5a724ad)

stephengillie commented:
> Retried package install using manifest and did not receive error. 

It's likely that our data center is blocked by IP address range. Do you have any way to contact with the developers?

noyard commented:
Yes I do.  can you provide what is needed to ask? If needed my email is noyard@microsoft.com 

here is the web page to download different versions
https://www.barbecana.com/full-monte/full-monte-download-msp/

stephengillie commented:
> Yes I do. can you provide what is needed to ask? If needed my email is [noyard@microsoft.com](mailto:noyard@microsoft.com)
> 
> here is the web page to download different versions https://www.barbecana.com/full-monte/full-monte-download-msp/

Sorry to miss this. What's needed is for someone to ask the developer to unblock our data center inside their routing infrastructure.

noyard committed:
Update IntraManagementSoftware&Tech.FullMonteMSP.locale.en-US.yaml

Stephen Gillie - I updated this package similar to your suggested edit to Schedule inspector :)

stephengillie commented:
Verify:
<img width="896" height="601" alt="image" src="https://github.com/user-attachments/assets/f7a20619-1b98-42c7-a20e-167cb7812704" />
<img width="457" height="164" alt="image" src="https://github.com/user-attachments/assets/5e00b971-567d-4f17-b25f-5e8ae948f364" />
